### PR TITLE
Add avator selector and new button styles

### PR DESCRIPTION
**Pull request by tjbo**
https://helpscout.atlassian.net/secure/RapidBoard.jspa?rapidView=153&projectKey=COMMS&modal=detail&selectedIssue=COMMS-283

**In this PR we are:**
1 - adding an AvatarSelector Component

2 - adding some styles to buttons to account for the pill shaped buttons

image

cc: @digitaldesigner

### CODE REVIEW

File: src/components/Button/styles/Button.config.css.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
+import { getColor } from '../../../styles/utilities/color'
```
Comment (tjbo) [inline src/components/Button/styles/Button.config.css.js]: The other file was getting difficult to read, so seperated this out.

File: src/components/AvatarSelector/AvatarSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  IconCaretUI,
+} from './styles/AvatarSelector.css.js'
+
+export interface Props {
+  image: string
+  isOpen: boolean
+  name: string
+}
+
+export default class AvatorSelector extends React.PureComponent<Props> {
```
Comment (tinkertrain) [inline src/components/AvatarSelector/AvatarSelector.tsx]: Little typo in class name "AvatorSelector"

File: src/components/AvatarSelector/AvatarSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  static default = {
```
Comment (tinkertrain) [inline src/components/AvatarSelector/AvatarSelector.tsx]: I think it should be `defaultProps` here

**Comment by tjbo** [comment]
Nice catch, fixed those up 👍

**Pull request merged**